test/images/regression-issue-74839/main.go

```diff
@@ -111,20 +125,26 @@ func probe(ip string) {
 				_, err := conn.WriteTo(badPkt.encode(localIP, remoteIP, data[:]), addr)
 				if err != nil {
 					log.Printf("conn.WriteTo() error: %v", err)
+				} else {
+					log.Println("boom packet injected")
 				}
 			}
 		}
 	}
 }
 
-func getIP() net.IP {
-	conn, err := net.Dial("udp", "8.8.8.8:53")
-	if err != nil {
-		log.Fatal(err)
+// getIPs gets the IPs from the downward API
+// we don't have to validate the IPs because
+// they are validated previously by kubernetes/CNI
+func getIPs() []net.IP {
+	var ips []net.IP
+	podIP, podIPs := os.Getenv("POD_IP"), os.Getenv("POD_IPS")
```

Comment:
It's okay to pass it in as env, but it will break the test, will it? also isn't `POD_IP` just a special case for `POD_IPS`?

Reply:
IIUIC the test will not pick this image until it is promoted to the registry,  and the version bumped in the manifest
That will be done in this PR https://github.com/kubernetes/kubernetes/pull/95351, that modifies the test accordingly and selects the new image version.

>  also isn't POD_IP just a special case for POD_IPS?
yeah, but that is only for newer versions (can't remember right now when it was added), but in order to support older version we need to keep checking Status.PodIP

Reply:
this gives preference to podips, but falls back to podIP if the former does not exist

Reply:
@anfernee any feedback on this?
I want to have this ready for the dual stack effort

Reply:
You don't need to support older kubernetes versions because they won't try to pull the new version of the issue-74839 image

Reply:
indeed that is true for k/k CI, but the feedback we get from KIND users is that there are still people using 1.12, 1.13, ...
This makes the image and the modified test https://github.com/kubernetes/kubernetes/pull/95351 that uses this image, able to run in older kubernetes versions without any cost.

Reply:
Sorry for the late reply. I am confused, when was `POD_IP` added? Isn't it to be added in the PR?

Reply:
POD_IP and POD_IPS are received as environment variables using the Downward API, this is the follow up PR with an example of the modified test using this new image https://github.com/kubernetes/kubernetes/pull/95351/files#diff-ad1f747fb42258e54abc48afe07f79ae44910b07376cd5671ff01e488900c7bcR296-R328.

In this PR we only modify the image and bump the version 1.1 -> 1.2, so current tests will keep using the image as it is today, 1.1.
Once the image 1.2 is in the registry, I will modify the test to use the new image and pass the POD_IP as environment variable. So, during some time, the current tests will be using the 1.1 image and the new test the 1.2